agenta-cli/agenta/docker/docker_utils.py

```diff
@@ -52,19 +52,18 @@ def build_tar_docker_container(folder: Path, file_name: Path) -> Path:
     shutil.copy(Path(__file__).parent / "docker-assets" / "entrypoint.sh", folder)
 
     # Read the contents of .gitignore file
-    gitignore_content = ""
-    gitignore_file_path = folder / ".gitignore"
-    if gitignore_file_path.exists():
-        with open(gitignore_file_path, "r") as gitignore_file:
-            gitignore_content = gitignore_file.read()
+    agentaignore_file_path = folder / ".agentaignore"
+    if agentaignore_file_path.exists():
+        with open(agentaignore_file_path, "r") as agentaignore_file:
+            agentaignore_content = agentaignore_file.read()
 
     # Create a temporary directory
     with tempfile.TemporaryDirectory() as temp_dir:
         temp_path = Path(temp_dir)
 
         # Clean - remove '/' from every files and folders in the gitignore contents
         sanitized_patterns = [
-            pattern.replace("/", "") for pattern in gitignore_content.splitlines()
+            pattern.replace("/", "") for pattern in agentaignore_content.splitlines()
```

Comment:
since we are not initializing the variable agentaignore_content (removed line 55), wouldn't that result an issue if .agentaignore does not exist?

Reply:
Most likely. However, it did work for me when I tested it. I added it to prevent any potential errors.